Symfony/CS/Tests/Fixer/Contrib/ShortEchoTagFixerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideClosingTagExamples
+     */
+    public function testOneLineFix($expected, $input = null)
+    {
```

Comment:
These tests need to be skipped on PHP 5.3 in case short open tags are disabled, because open tags with echo are disabled as well by this flag on 5.3